### PR TITLE
Cxtb 261 automate onboarders access facilities directory%filter

### PR DESCRIPTION
**Pull request by jmontero-nts**
Adding automated test of an onboarded account when accessing facilities.

### CODE REVIEW

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml [hunk 1]
```diff
@@ -62,4 +62,95 @@ TestManuallyLogOutOfThePortalOnboarder:
...
+     Id: $PAGE.care_platform_participant.first_senior_lastname$
+  #Click on an user
+   - Type: click
+     Strategy: xpath
+     Id: $PAGE.care_platform_participant.senior_1_row$
+   - Type : wait_for
+     Strategy: xpath
+     Id: $PAGE.care_platform_facility.facility_name$
+  #Verify that a summary component is displayed
+
```
Comment (miguel-hquez) [inline tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml:155]: is there any steps for this summary component is displayed?

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml [hunk 2]
```diff
@@ -62,4 +62,95 @@ TestManuallyLogOutOfThePortalOnboarder:
     Id: $PAGE.care_platform_logout_modal.cancel_logout_button$
   # Repeat the steps from above and log out.
   - Type: case
-    Value: CarePlatformLogout
\ No newline at end of file
+    Value: CarePlatformLogout
```
Comment (miguel-hquez) [inline tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml:65]: Please add a breakline between tests to keep format

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml (hunk 1 above)
Comment (jmontero-nts) [reply tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml:154]: I deleted it. there is not any step to be displayed..

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml (hunk 2 above)
Comment (jmontero-nts) [reply tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml:65]: Done

File: tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml [hunk 3]
```diff
@@ -62,4 +62,95 @@ TestManuallyLogOutOfThePortalOnboarder:
...
+# CXTB-261-Access facility Directory & Filter
```
[the remark this answers is not in the document]
Comment (jmontero-nts) [reply tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml]: Done, this comments is resolved by this commit https://github.com/neveralone-chs-org/TestRay/pull/86/commits/8214f1f98f020a723ecf3f455c0b20744558b1a5

**Review by csanchez-nts — APPROVED** [review]

**Review by miguel-hquez — APPROVED** [review]

**Pull request merged**